packages/react-native-web/package.json

```diff
@@ -26,7 +26,9 @@
     "create-react-class": "^15.7.0",
     "fbjs": "^3.0.4",
     "inline-style-prefixer": "^6.0.1",
+    "memoize-one": "^6.0.0",
     "normalize-css-color": "^1.0.2",
+    "nullthrows": "^1.1.1",
```

Comment:
I didn't see where this gets used

Reply:
[Here](https://github.com/necolas/react-native-web/blob/928637f7548af04e57e2a6378398171284e7f7f5/packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js#L1597), inside VirtualizedList/index.js